airflow/models.py

```diff
@@ -116,7 +116,7 @@ def get_fernet():
         raise AirflowException("Could not create Fernet object: {}".format(ve))
 
 
-if 'mysql' in settings.SQL_ALCHEMY_CONN:
+if settings.SQL_ALCHEMY_CONN and 'mysql' in settings.SQL_ALCHEMY_CONN:
```

Comment:
It looks like LongText is no longer being used in any of the models, so we can remove this section